runtests.py

```diff
@@ -610,8 +614,10 @@ def handle_directory(self, path, context):
                     test_class = CythonUnitTestCase
                 else:
                     test_class = CythonRunTestCase
-            else:
+            elif mode in ['compile', 'error', 'test']:
```

Comment:
Is there really a 'test' test mode?

Reply:
Yes, you can see it on the first failed build of this PR